test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestBinder.cs

```diff
@@ -58,5 +58,13 @@ public Type GetReturnType(string invocationId)
             }
             throw new InvalidOperationException("Unexpected binder call");
         }
+
+        public Type GetStreamItemType(string streamId)
+        {
+            // in signalR 2, stream items were only sent from server -> client
```

Comment:
in signalR 2

->

in v1

("signalR 2" can be confused for the other SignalR, ASP.NET SignalR ;))